en/best-practices.md

```diff
@@ -70,6 +70,7 @@ __Examples:__
 
 | Field Name | Recommendation |
 | --- | --- |
+| `agency_id` | Must be included if it is defined in `agency.txt`. |
```

Comment:
@isabelle-dr Rather than using the word `must`, should this match the language used in [agency.txt](https://github.com/MobilityData/GTFS_Schedule_Best-Practices/pull/52/files#diff-8f901be24428812f860668f5dbc2b94d6728a2135297940981416b6433fc2569R46)?
```markdown
| `agency_id` | Should be included, even if there is only one agency in the feed. (See also recommendation to include `agency_id` in [`routes.txt`](#routestxt) and [`fare_attributes.txt`](#fare_attributestxt)) |
```

Also regarding the implementation [#878](https://github.com/MobilityData/gtfs-validator/issues/878) should we add similar validation with warning/error depending on one or more agencies in `agency.txt`?